### PR TITLE
chore(ci): disable ES5 generation since it slows down the build

### PR DESCRIPTION
**Pull request by arturovt**

### CODE REVIEW

**Comment by hoebbelsB** [comment]
this is a cool idea! the demo app isn't hosted anywhere, is it?

File: apps/docs/.browserslistrc [hunk 1]
```diff
@@ -5,8 +5,4 @@
 # You can see what browsers were selected by your queries by running:
 #   npx browserslist
 
-> 0.5%
```
Comment (hoebbelsB) [inline apps/docs/.browserslistrc:8]: @Karnaukhov-kh will this be a problem for our docs page?

File: apps/ssr/.browserslistrc [hunk 2]
```diff
@@ -9,9 +9,3 @@
 #   npx browserslist
 
 last 1 Chrome version
-last 1 Firefox version
```
Comment (hoebbelsB) [inline apps/ssr/.browserslistrc:12]: @edbzn is this fine for the ssr setup or do we require having those settings here?

**Review by hoebbelsB — COMMENTED** [review]
great idea, just left comments where I think we should have a second look at

File: apps/ssr/.browserslistrc (hunk 2 above)
Comment (edbzn) [reply apps/ssr/.browserslistrc:12]: It's ok.

File: apps/docs/.browserslistrc (hunk 1 above)
Comment (arturovt) [reply apps/docs/.browserslistrc:8]: The docs page is not an Angular app.

**Comment by BioPhoton** [comment]
Other opinions on dropping IE?

**Comment by Karnaukhov-kh** [comment]
In the Readme we are saying that we support IE. This is true for state but not true for template, because we still have globalThis and it requires a polyfill for IE and old Edge.

Dropping IE and old Edge sounds good but maybe we need to have a separate browser support for state and template then.

**Comment by arturovt** [comment]
Well, this change actually related to apps in our repository but not to libraries, `ng-packagr` doesn't use browserlistrc configuration.

**Comment by edbzn** [comment]
This PR is about demos, ssr, etc.. which are not production apps, it will not affect library distribution bundles.

> In the Readme we are saying that we support IE. This is true for state but not true for template, because we still have globalThis and it requires a polyfill for IE and old Edge.

True, this is confusing.

**Comment by Karnaukhov-kh** [comment]
Ok, no objections from my side then :)

**Pull request merged**